src/components/Toast/__tests__/toast-tests.jest.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { fireEvent, render, cleanup } from "@testing-library/react";
+import { act } from "@testing-library/react-hooks";
+import Toast from "../Toast";
+
+const renderComponent = props => {
+  return render(<Toast {...props} />);
+};
+
+describe("Toast tests", () => {
```

Comment:
please add test for calling onClick when clicking on attached button to the toast (by using the actions prop)